README.md

```diff
@@ -1,2 +1,53 @@
-# bc-cms-rewrite-node-ms
-Node.js layer that serves api to front end and fetches data from Mongo (Backend for Frontend)
+# Express API Starter
+
+How to use this template:
```

Comment:
Add some meaningful details of this repo and keep updating it consistently for the future changes also.